dotnet/dotnet-cli/Portfile

```diff
@@ -31,15 +52,15 @@ master_sites        https://dotnetcli.azureedge.net/dotnet/Runtime/${version}/
 switch ${build_arch} {
     x86_64 {
         distname            dotnet-runtime-${version}-osx-x64
-        checksums           rmd160  37f25257f3d088a77c1abd87a4dc6b289fc107ba \
-                            sha256  b0b6a680f196c6ec14387b1968bec2d3e59fe0c88f6906270b4bedcbb64d852a \
-                            size    30847546
+        checksums           rmd160  4059918e8dfa0e40180589363d54ebe46acb84e1 \
+                            sha256  5d4847009e78846278a5ed83cef3a367a02e274e4582a5c89259810e7146bf9e \
+                            size    30866950
     }
     arm64 {
         distname            dotnet-runtime-${version}-osx-arm64
-        checksums           rmd160  486342ae4505bb501c40bd929668b91ff162b0e2 \
-                            sha256  cbd2cd5825e6d55155d0293e2bf0c08fd0a206f4c5a939690b087c76d18b6d50 \
-                            size    29178452
+        checksums           rmd160  8e9024427b4a7426060afd8cdb36ee9db04656f0 \
+                            sha256  43a129205152389a201d921fcf14565a316bc2a6a271354a08879ad5e4f9d990 \
+                            size    29186244
```

Comment:
@silverbzez probably a copy-paste error here, see: https://trac.macports.org/ticket/68580

Reply:
Oops my bad, will fix it right away.